### PR TITLE
TC 14.1.2_07 Verify that when you select each category in the "Style" option, product cards are displayed on the page in the quantity available for this category

### PR DESCRIPTION
**Pull request by teya-teya**
https://trello.com/c/ARC6VeI3

### CODE REVIEW

File: helpers/testData.js [hunk 1]
```diff
@@ -83,6 +83,7 @@ export const EMAIL = generateRandomEmail();
 export const MY_ACCOUNT_HEADER = 'My Account';
 export const THANKS_MESSAGE = 'Thank you for registering with Main Website Store.';
 export const EXPECTED_ITEM_STYLE_WOMEN_BOTTOMS = ['Base Layer', 'Basic', 'Capri', 'Compression', 'Leggings', 'Parachute', 'Snug', 'Sweatpants', 'Track Pants'];
+export const PRODUCT_CARDS = '.item.product.product-item';
```
[the remark this answers is not in the document]
Comment (teya-teya) [reply helpers/testData.js:86]: It's not a locator, it's a text value that I put in a variable

[the remark this answers is not in the document]
Comment (teya-teya) [reply helpers/testData.js:86]: Sorry, the page was not updated and I accidentally clicked on repeat

File: page_objects/bottomsWomenPage.js [hunk 2]
```diff
@@ -16,7 +16,17 @@ class BottomsWomenPage {
         getOptionPrice: () => this.page.locator('.filter-options-title').nth(3),
         getOptionPriceFilter: () => this.page.locator('.filter-options-content').nth(3),
         getCategoriesStyle: () => this.page.$$('a[href*=\'style\']'),
-        getCountItemsInCategoryStyle: (category) => category.$('span.count')
+        getCountItemsInCategoryStyle: (category) => category.$('span.count'),
+        getCountsItemsInCategoryStyle: () => this.page.$$("a[href*='style']>span.count"),
+        getObjectCategoriesStyle: () => this.page.$$eval("a[href*='style']", elements =>
```
Comment (SvetlanaKanel) [inline page_objects/bottomsWomenPage.js:21]: this is not a locator, this is a method that returns name and count. You need to add to locators only locators and write your method under all methods that already exist.

File: tests/test_page_objects/bottomsWomenPage.spec.js [hunk 3]
```diff
@@ -56,6 +56,24 @@ test.describe('bottomsWomenPage.spec', () => {
...
+        
+        await homePage.hoverWomenMenuitem();
+        await homePage.clickBottomsWomenLink();
+    
+        await expect(page).toHaveURL(BASE_URL + BOTTOMS_WOMEN_PAGE_END_POINT);
+    
+        const categoriesStyle = await bottomsWomenPage.locators.getObjectCategoriesStyle();
+        const productCards = PRODUCT_CARDS;
+        
+        await bottomsWomenPage.verifyProductCardsQuantity(categoriesStyle, productCards);
```
Comment (SvetlanaKanel) [inline tests/test_page_objects/bottomsWomenPage.spec.js:72]: I cannot see the expected checkings of the results that you mention in the name of your test. You hidded all checks to your method but they must be in the test.

File: page_objects/bottomsWomenPage.js (hunk 2 above)
Comment (teya-teya) [reply page_objects/bottomsWomenPage.js]: fixed it

**Pull request closed without merging**